docs/USAGE-GUIDE.md

```diff
@@ -912,6 +912,16 @@ extern s2n_cert_private_key *s2n_cert_chain_and_key_get_private_key(struct s2n_c
 **s2n_cert_chain_and_key_get_private_key** returns a private key from
 **s2n_cert_chain_and_key** object.
 
+### s2n\_config\_enable\_cert\_req\_dss\_legacy\_compat
+
+```c
+extern int s2n_config_enable_cert_req_dss_legacy_compat(struct s2n_config *config);
+```
+
+**s2n_config_enable_cert_req_dss_legacy_compat** adds a dss cert type in the server certificate request when being called. 
+It only sends the dss cert type in the cert request but does not succeed the handshake if a dss cert is received. 
+Please DO NOT call this api unless you really understand what it does
+
```

Comment:
Sorry I should have been more specific about the documentation on this. I don't think we want it to be officially documented so let's just put this in a comment before the function in the `api.h`, rather than making it here.

Reply:
Gotcha